pages/index.js

```diff
@@ -54,13 +54,13 @@ function HomePage() {
           </Paragraph>
           <ul className="md:grid md:gap-8 md:grid-cols-2 lg:grid-cols-5">
             <li className="justify-center flex">
-              <AdidasLogo className="h-8 text-gray-400" />
+              <AdidasLogo className="h-8 text-gray-400 hover:text-black" />
             </li>
             <li className="justify-center flex">
-              <AxwayLogo className="mt-12 md:-mt-5 lg:-mt-5 h-14 text-gray-400" />
+              <AxwayLogo className="mt-12 md:-mt-5 lg:-mt-5 h-14 text-gray-400 group hover:text-black" />
             </li>
             <li className="justify-center flex">
-              <SlackLogo className="mt-12 md:mt-2 lg:-mt-1 h-10 text-gray-400" />
+              <SlackLogo className="mt-12 md:mt-2 lg:-mt-1 h-10 text-gray-400 hover:text-black group" />
```

Comment:
can't we do it somehow in a consistent way, so `hover:text-black group` is everywhere?

it might be a dump question, sorry, but just want to make sure we can be consistent on component level instead of introducing changes in SVG code (unless it is just simply not possible)

Reply:
Actually all SVGs have diffrent color codes in them, so we have to manually edit the SVGs code for the logos.